bylaws.tex

```diff
@@ -0,0 +1,288 @@
+
+\section{Quorum}
+
+Four Directors present in person shall constitute a quorum for the transaction of business at any meeting of the Board of Directors, unless the presence of a greater number is required by law, in which case the quorum shall be such number as the law requires.
+
+\section{Manner of Acting}
+
+\subsection{Formal Action by Directors}
+
+The act of a majority of Directors present at a meeting at which a quorum is present shall be the act of the Board of Directors.
+
+\subsection{Informal Action by Directors}
+
+No action of the Board of Directors shall be valid unless taken at a meeting at which a quorum is present, excepting any action taken without a meeting for which a consent in writing, setting forth the action so taken, is signed by each Director entitled to vote.
+
+\section{Resignations and Removal}
+
+Any elected Director may resign from the Board of Directors at any time by giving notice in writing to the President or Secretary of the Corporation, and unless otherwise specified therein, the acceptance of such resignation shall not be necessary to make it effective.
+
+Additionally, any elected Director who is absent from all meetings of the Board of Directors occurring in a thirteen month period entirely within his term of office, the number of meetings in the period for which notice was properly given being at least two, thereby resigns from the Board of Directors.
+
+\section{Vacancies}
+
+Any vacancy occurring in an elected Directorship may be filled by the Board of Directors for the unexpired portion of the term of office. Any Directorship to be filled by reason of an increase in the number of Directors shall be filled by the Board of Directors.
+
+\section{Compensation}
+
+Directors shall not receive any stated salaries for their services but, by resolution of the Board of Directors, expenses of attendance may be allowed for attendance at each regular or special meeting of the Board of Directors.
+
+\section{Procedure}
+
+The Chairman of the Board of Directors, or in his absence, the President, shall preside at meetings of the Board of Directors. The Board of Directors may adopt its own rules of procedure which shall not be inconsistent with these Bylaws.
+
+\chapter{Officers}
+
+\section{Officers}
+
+The officers of the Corporation shall be a President, a Vice President, a Secretary, and a Treasurer. The Corporation may, at the direction of the Board of Directors, have additional officers including, without limitation, other vice presidents, one or more assistant secretaries, and one or more assistant treasurers. One person may hold two or more offices except one person may not hold both the offices of President and Secretary.
```

Comment:
One person shouldn't hold president and treasurer as well I think

Reply:
ok